op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -147,21 +150,18 @@ func Test_ProgramAction_OperatorFeeConstistency(gt *testing.T) {
 			setStorageInUpdateContractTo(1)
 			rSet := env.Alice.L2.LastTxReceipt(t)
 			require.Equal(t, uint64(43696), rSet.GasUsed)
-			aliceInitialBalance, l1FeeVaultInitialBalance, baseFeeVaultInitialBalance, sequencerFeeVaultInitialBalance, operatorFeeVaultInitialBalance = getCurrentBalances()
+			aliceInitialBalance, l1FeeVaultInitialBalance, baseFeeVaultInitialBalance, sequencerFeeVaultInitialBalance, operatorFeeVaultInitialBalance, depositorInitialBalance = getCurrentBalances()
 			setStorageInUpdateContractTo(0)
 			rUnset := env.Alice.L2.LastTxReceipt(t)
 			// we assert on the exact gas used to show that a refund is happening
 			require.Equal(t, uint64(21784), rUnset.GasUsed)
 
 		case DepositTx:
-			aliceInitialBalance, l1FeeVaultInitialBalance, baseFeeVaultInitialBalance, sequencerFeeVaultInitialBalance, operatorFeeVaultInitialBalance = getCurrentBalances()
-
-			bobInitialBalance := balanceAt(env.Bob.Address())
+			aliceInitialBalance, l1FeeVaultInitialBalance, baseFeeVaultInitialBalance, sequencerFeeVaultInitialBalance, operatorFeeVaultInitialBalance, depositorInitialBalance = getCurrentBalances()
 
 			// regular Deposit, in new L1 block
 			env.Alice.L1.ActResetTxOpts(t)
 			env.Alice.L2.ActSetTxToAddr(&env.Dp.Addresses.Bob)(t)
-			env.Alice.L2.ActSetTxValue(new(big.Int).SetUint64(testDepositValue))(t)
```

Comment:
I think we actually want to keep this test case as well, a deposit with a value. This PR should probably add a new case here that doesn't include a value.

Reply:
What new case would we cover with this? We also don't transfer values in the other cases. I'm not sure it's that important since what we want to cover in the fee tests are tx gas usage.

Reply:
That's true, will resolve.